great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -0,0 +1,212 @@
+                parameters=parameters,
+            )
+        else:
+            metric_value_kwargs = self._metric_value_kwargs
+
+        expectation_suite_name: str = f"tmp_suite_domain_{domain.id}"
+        validator_for_metrics_calculations: Validator = self.data_context.get_validator(
+            batch_request=self.batch_request,
+            create_expectation_suite_with_name=expectation_suite_name,
+        )
+
+        metric_values: List[Union[float, np.float32, np.float64]] = []
+        metric_domain_kwargs_with_specific_batch_id: Optional[
+            Dict[str, Any]
+        ] = copy.deepcopy(metric_domain_kwargs)
+        batch_id: str
+        for batch_id in batch_ids:
+            metric_domain_kwargs_with_specific_batch_id["batch_id"] = batch_id
+            metric_configuration_arguments: Dict[str, Any] = {
+                "metric_name": self._metric_name,
+                "metric_domain_kwargs": metric_domain_kwargs_with_specific_batch_id,
+                "metric_value_kwargs": metric_value_kwargs,
+                "metric_dependencies": None,
+            }
+            metric_value: Union[
+                Any, float
+            ] = validator_for_metrics_calculations.get_metric(
+                metric=MetricConfiguration(**metric_configuration_arguments)
+            )
+
+            if not is_numeric(value=metric_value):
+                raise ge_exceptions.ProfilerExecutionError(
+                    message=f"""Applicability of {self.__class__.__name__} is restricted to numeric-valued metrics \
+(value of type "{str(type(metric_value))}" was computed).
+"""
+                )
+
+            metric_values.append(float(metric_value))
+
+        metric_values = np.array(metric_values, dtype=np.float64)
+        mean: float = np.mean(metric_values)
+        std: float = np.std(metric_values)
+
+        num_stds: Union[float, int] = NP_SQRT_2 * special.erfinv(
+            1.0 - self._false_positive_rate
+        )
+        num_stds = float(np.rint(num_stds))
+
+        min_value: float = mean - num_stds * std
+        max_value: float = mean + num_stds * std
+
+        parameter_values: Dict[str, Any] = {
+            self.fully_qualified_parameter_name: {
+                "value": {
+                    "min_value": min_value,
+                    "max_value": max_value,
+                },
+                "details": {
+                    "metric_configuration": {
+                        "metric_name": self._metric_name,
+                        "metric_domain_kwargs": metric_domain_kwargs,
+                        "metric_value_kwargs": metric_value_kwargs,
+                        "metric_dependencies": None,
+                    },
+                },
+            },
+        }
+
+        build_parameter_container(
+            parameter_container=parameter_container, parameter_values=parameter_values
+        )
+
+    @property
+    def fully_qualified_parameter_name(self) -> str:
```

Comment:
We decided to remove this `fully_qualified_parameter_name` and use explicit `max_value` and `min_value` 
For example to reference in ExpectationConfigurationBuilder (where row_count is the ParameterBuilder name):
```
min_value: $parameter.row_count.value.min_value
max_value: $parameter.row_count.value.max_value
```

Reply:
Alex plans to remove the fully_qualified_parameter_name property in a follow-up PR.

Reply:
Done -- in next commit.